.travis.yml

```diff
@@ -9,8 +9,8 @@ before_script:
 install:
   - npm install
 script:
+  - npm run link
   - npm run lint
   - npm run test
-  - npm run build
```

Comment:
The `build` script is automatically ran after install because of [npm prepare](https://docs.npmjs.com/misc/scripts)